Dockerfile

```diff
@@ -86,15 +84,18 @@ WORKDIR /app
 COPY ./bin/docker_start.sh /start.sh
 COPY ./bin/wait_for_db.sh /wait_for_db.sh
 COPY ./bin/celery_worker.sh /celery_worker.sh
-COPY ./bin/check_celery_worker_liveness.py ./bin/
+COPY ./bin/celery_beat.sh /celery_beat.sh
 COPY ./bin/setup_configuration.sh /setup_configuration.sh
-RUN mkdir /app/log
-RUN mkdir /app/media
+RUN mkdir /app/log /app/media /app/private_media /app/tmp
+COPY ./bin/check_celery_worker_liveness.py ./bin/
+
+# prevent writing to the container layer, which would degrade performance.
+# This also serves as a hint for the intended volumes.
+VOLUME ["/app/log", "/app/media", "/app/private_media"]
 
 # copy backend build deps
 COPY --from=backend-build /usr/local/lib/python3.11 /usr/local/lib/python3.11
 COPY --from=backend-build /usr/local/bin/uwsgi /usr/local/bin/uwsgi
-COPY --from=backend-build /app/src/ /app/src/
```

Comment:
I suspect this will break because OIP uses non-PyPI dependencies: https://github.com/maykinmedia/open-inwoner/blob/574199acc11fb89785386043e7f30ee732f08ac6/requirements/base.in#L91

Reply:
Hmm, Docker build worked locally and the image is built in the CI. Are there other things I need to worry about?

Reply:
not that I can tell right now, but surprises may happen of course due to project specifics :P

Reply:
I remember what the deal is - when you `pip install -e ...`, then the code goes into `/app/src` so that's why it had to be copied over. While OIP uses non-PyPI dependencies, they aren't being installed in editable mode and go into site-packages, so this COPY instruction is now indeed obsolete.